docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
 or there can be naming conventions, e.g. the event list for run ``42`` could be stored
 at ``$TEV_DATA/RUN_000042/Events.fits``.
 
+A run list must have at least a column called ``OBS_ID``::
+ 
+   OBS_ID
+   42
+   43
 
-CSV format
-----------
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is:
 
-We use the `CSV <http://en.wikipedia.org/wiki/Comma-separated_values>`_ (comma-separated values) format for run lists.
-This has the advantage that everyone can work with whatever tool they like. Here's some good options:
+================  ================================================================================================  =========
+column name       description                                                                                       required?
+================  ================================================================================================  =========
+OBS_ID            observation ID as an integer                                                                      yes
+RA                pointing position right ascension in equatorial (ICRS) coordinates                                yes?
+DEC               pointing position declination in equatorial (ICRS) coordinates                                    yes?
+AZ                average azimuth angle during the observation                                                      yes?
+ALT               average altitude angle during the observation                                                     yes?
+MUON_EFFICIENCY   average muon efficiency of the telescopes                                                         yes?
+TIME_START        start time of the observation stored as number of seconds after the time reference in the header  yes?
+TIME_STOP         end time of the observation in the same forma as TIME_START                                       yes?
+TIME_OBSERVATION  duration of the observation                                                                       yes?
+TIME_LIVE         duration of the observation without dead time                                                     yes?
+TRIGGER_RATE      average trigger rate of the system                                                                no
+MEAN_TEMPERATURE  average temperature of the atmosphere                                                             no
+N_TELS            number of telescopes participating in the observation                                             yes?
+TEL_LIST          string with a CSV list of telescope IDs participating in the observation                          yes?
+QUALITY           data quality; recommended: "spectral" or "detection" (not used by Gammapy at the moment)          no
```

Comment:
Remove "(not used by Gammapy at the moment)". There's more things like `TRIGGER_RATE`, ... that aren't used, no need to try and convey that info here.